src/components/GoalActivity.tsx

```diff
@@ -25,7 +26,7 @@ interface GoalActivityProps {
     header?: React.ReactNode;
     footer?: React.ReactNode;
 
-    renderCommentItem: (item: NonNullable<GoalByIdReturnType>['comments'][number]) => React.ReactNode;
+    renderCommentItem: (item: GoalComment) => React.ReactNode;
```

Comment:
Why not CommentProps here?

Reply:
we can do this separately https://github.com/taskany-inc/issues/issues/1955